sw/device/silicon_creator/lib/drivers/spi_device.c

```diff
@@ -316,13 +327,13 @@ static_assert(kBfptTablePointer % sizeof(uint32_t) == 0,
  * BFPT 16th Word
  * --------------
  * [31:14]: 4-Byte addressing (not supported, 0x0)
- * [13: 8]: Soft-reset (instruction 0xf0, 0x80)
+ * [13: 8]: Soft-reset (0x66/0x99 sequence, 0x10)
```

Comment:
spi_device is actually incapable of proving that the sequence happened. Even with space in the command FIFO, we are only able to track the order of uploaded commands. If a read came in between RSTEN and RST, firmware has no way to know (since we can't really handle uploading read commands, heh).

It's not the end of the world. Soft reset is one of those things that isn't necessarily implemented and isn't a critical feature, so if ours doesn't perfectly match, there's not a significant impact. I just wish we weren't advertising support for a specific standard feature in the SFDP if we aren't doing it to the letter. /shrug

Reply:
Ack, thanks @a-will! Discussed this with cfrantz offline, decided to not enforce RSTEN for the sake of simplicity.